src/accelerate/commands/launch.py

```diff
@@ -98,7 +98,7 @@ def __call__(self, parser, namespace, values, option_string=None):
             "optional arguments",
         ]
         if len(args) > 1:
-            used_platforms = [arg for arg in args if arg in options_to_group.keys()]
+            used_platforms = [arg for arg in args if arg in options_to_group]
```

Comment:
Disagree with this change here. It's more readable to explicitly have the `.keys()`.